include/deal.II/base/timer.h

```diff
@@ -574,6 +574,32 @@ class Timer
  * sure that we only generate output on a single processor. See the step-32,
  * step-40, and step-42 tutorial programs for this kind of usage of this class.
  *
+ * The second variant to cope with this issue is print more information about
+ * the recorded times to be able to understand this kind of imbalances without
+ * actually adding the barriers. While this approach is still affected by
+ * imbalances between different MPI processes, its output is not the arbitrary
+ * time of rank 0, but the minimum, average and maximum of the MPI results,
+ * using information from Utilities::MPI::MinMaxAvg. As the data is also
+ * equipped with the rank id where the minimum and maximum are attained, this
+ * approach allows to identify on which ranks certain slowdowns occur. In case
+ * some imbalance between the MPI ranks from one section to the next can be
+ * tolerated, this strategy can hence be advantageous over the barrier variant
+ * as it does not synchronize the program in places where it is not
+ * necessary. The usage of this variant is to initialize the output object
+ * without any native print settings and without communicator,
+ * @code
+ *   TimerOutput timer (pcout,
+ *                      TimerOutput::never,
+ *                      TimerOutput::wall_times);
+ * @endcode
+ * and then call
+ * @code
+ *   timer.print_summary_statistics(MPI_COMM_WORLD);
+ * @endcode
+ * Here, the output is written to the <code>pcout</code> object of type
+ * ConditionalOStream passed to the constructor, making sure the information
+ * is only printed once. See step-67 for an example usage of this variant.
```

Comment:
I will most definitely change step-69 to use the new variant as well :-)

Reply:
> I will most definitely change step-69 to use the new variant as well :-)

The second order variant of step-69 was one of the applications where this has bothered me - in addition to many of my large-scale scaling tests where I wanted the imbalance to be propagated between different timer sections and visualize it.